### PR TITLE
Change properties displayed for top 5 projects on DimagiSphere

### PR DESCRIPTION
**Pull request by rickypchen**
Per feedback from Gillian, when user clicks on a country, it will display top five most active projects based on the number of active mobile users with the following properties: 
- organization name
- sector
- deployment date
- number of active users
- countries

@czue

### CODE REVIEW

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js [hunk 1]
```diff
@@ -210,6 +210,18 @@ var projectMapInit = function(mapboxAccessToken) {
...
+    function capitalizeCountryNames(countries) {
+        return countries.map(function(country) {
+            var formattedCountryName = country.charAt(0).toUpperCase();
+            if (country.indexOf(",") > -1) {
+                formattedCountryName += country.substring(1, country.indexOf(",")).toLowerCase();
+            } else {
+                formattedCountryName += country.substring(1).toLowerCase();
+            }
+            return formattedCountryName;
+        })
```
Comment (dimagimon) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: Missing semicolon. (semi)

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js [hunk 2]
```diff
@@ -210,6 +210,18 @@ var projectMapInit = function(mapboxAccessToken) {
         info.update();
     }
 
+    function capitalizeCountryNames(countries) {
+        return countries.map(function(country) {
+            var formattedCountryName = country.charAt(0).toUpperCase();
+            if (country.indexOf(",") > -1) {
```
Comment (czue) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: what case is this handling? very nit picky but if we're also cleaning / truncating some names the method name is a little misleading (maybe switch to `formatCountryNames`?)

**Comment by czue** [comment]
nice this is a good call - we were just looking at this on friday and had the same recommendation in terms of ordering and adding active users.

File: corehq/apps/hqadmin/static/hqadmin/js/project_map.js (hunk 2 above)
Comment (rickypchen) [inline corehq/apps/hqadmin/static/hqadmin/js/project_map.js]: @czue I agree. This is for formatting an array of country names that are in all caps, so it is a little misleading to say `capitalizeCountryNames`. I'll switch it back to `formatCountryNames`

**Pull request merged**